integration/prefetch-test.ts

```diff
@@ -166,7 +166,8 @@ test.describe("prefetch", () => {
           await page.waitForSelector(
             // Look for either Rollup or Rolldown chunks
             [
-              "#nav link[rel='modulepreload'][href^='/assets/chunk-']",
+              // Rollup bundles RR (and transitively React) into an index chunk
+              "#nav link[rel='modulepreload'][href^='/assets/index-']",
```

Comment:
@markdalgleish Does this feel expected or unexpected to you now that I added the external config to ESM?  I'm temped to remove it so we stick with the prior ESM handling...

Reply:
As in, this change is introducing an additional chunk in the final app build?